build.ps1

```diff
@@ -22,7 +24,7 @@ function bazel() {
 }
 
 function build-partial() {
-    bazel build `
+    bazel build --experimental_execution_log_file ${ARTIFACT_DIRS}/build_partial_execution_windows.log `
```

Comment:
hyphens might need escaping via backticks - see `bazel run` below

Reply:
Thanks! I've escaped the hyphens.